target.c

```diff
@@ -329,8 +330,13 @@ int tcmu_add_dev_to_recovery_list(struct tcmu_device *dev)
 		ret = -ENOMEM;
 		goto done;
 	}
+
+	pthread_attr_init(&attr);
+	pthread_attr_setdetachstate(&attr, PTHREAD_CREATE_DETACHED);
```

Comment:
The `attr` here is defined not used, we should `s/NULL/attr/g` in `pthread_create()`.